### DISCUSSION TITLE
Do staged files continue to be updated if I edit the working files?<title>

### DISCUSSION THREAD

**Discussion by KateBellman** [community/community#60827] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

If I've staged a file, but then I continue to edit that file, does that staged file change or has it taken a snapshot of that file when I staged it?

**Comment by the-officialjosh** [comment, marked as answer]
Once a file is staged, it does not automatically update with further edits. The staged file represents the state of the file at the time of staging. To include the new changes, you need to stage the file again.

**Reply by KateBellman** [reply]
Ah ok, that makes sense. Thanks!
